src/effects/effectchainslot.cpp

```diff
@@ -390,3 +391,51 @@ void EffectChainSlot::slotChannelStatusChanged(const QString& group) {
 unsigned int EffectChainSlot::getChainSlotNumber() const {
     return m_iChainSlotNumber;
 }
+
+QDomElement EffectChainSlot::toXML(QDomDocument* doc) const {
+    QDomElement chainElement = doc->createElement("EffectChain");
+    if (m_pEffectChain == nullptr) {
+        return chainElement;
+    }
+
+    XmlParse::addElement(*doc, chainElement, "Name",
```

Comment:
Anonymous namespaces will not work well for this because these strings are reused in both the Slot (toXML & loadValuesFromXml) and non-Slot (createFromXml) classes in src/effects. Can you suggest a better place to define the strings?